ResponsiveBlocks.html

```diff
@@ -0,0 +1,77 @@
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=10">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Lato" />
+  <link rel="stylesheet" href="new.css">
+</head>
+
+<body>
+  <div id='stars'></div>
+  <div id='stars2'></div>
+  <div id='stars3'></div>
+
+
+
+
+
+
+  
+    <div class="col-md-10 col-md-offset-1">
+        <div class="rectangle">
+          
+                        
+              
+              <div class="col-md-2 col-sm-2">
+                
+                <div class="rect">
+              <img src="bee.png" alt="" class="image">
```

Comment:
Fix indentation. Install an HTML linter in your editor.

Reply:
Done